lib/puppet/type/firewalld_ipset.rb

```diff
@@ -13,6 +11,10 @@
         }
   "
 
+  def po2?(n)
```

Comment:
Technically, this should be made private but it doesn't matter that much.

Reply:
Making it `private` didn't go so well.

> expected Puppet::Error with message matching /hashsize must be a power of 2/, got #<Puppet::ResourceError: Parameter hashsize failed on Firewalld_ipset[test]: Validate method failed f...lass hashsize: private method `po2?' called for #<Puppet::Type::Firewalld_ipset:0x000055bb0f7f56c8>>